addons/mail/static/src/scss/opacity.scss

```diff
@@ -0,0 +1,7 @@
+// .opacity-hover class disable the opacity effect (see opacity-100-hover) on
+// device that dont support the hover effect (mainly touch device).
+@media (hover: none) {
+    .opacity-hover {
+        opacity: 100 !important;
+    }
+}
```

Comment:
I would at least prefix the classname with `o-`, to clarify this is a odoo custom classname.
BS classname (including extensions) without `o-` prefix are at the discretion of RD design team.

Also a new SCSS file is overkill. You could just dump it in `core/common/core.scss`. This file is expected to be loaded at the correct time. Don't use the old folder convention `src/scss/` that was a bad split in folder and also do not work well with the conventional split in folder (which has a purpose, so following it to the letter is important).

Lastly, definitely a nitpick, I would probably name it `o-opacity-hoverable`. This is because `opacity-hover` looks too close to `opacity-100-hover` with lacking `100`, so people will likely misunderstand the semantics of it. Using `hoverable` should be more intuitive to anyone.

Reply:
I agree with all that and made the changes